### PR TITLE
Changes to allow to run triplo deployment from openstackclient pod

### PR DESCRIPTION
**Pull request by stuggi**
- connects openstackclient to OSP network using static ipam. Right
now hard coded ip 192.168.25.6
- tripleo templates + deploy script get mounted from configmaps
- the configmaps are atm prepared using the osp-director-dev-tools
playbooks
- overcloud can be installed using:
$ source /root/ostest-working/oc_env.sh
$ rsh openstackclient
$ /scripts/tripleo-deploy.sh

### CODE REVIEW

File: controllers/controllervm_controller.go [hunk 1]
```diff
@@ -129,7 +129,7 @@ func (r *ControllerVMReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 			Namespace:      instance.Namespace,
 			Type:           common.TemplateTypeConfig,
```
Comment (abays) [inline controllers/controllervm_controller.go:130]: Should this be `common.TemplateTypeNone`, as per our conversation last week?  I knew you said you were adding a new type, but I know this works anyhow.

File: controllers/controllervm_controller.go [hunk 2]
```diff
@@ -142,6 +142,41 @@ func (r *ControllerVMReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
...
+				DomainNameUniq:    fmt.Sprintf("%s-%s", hostname, instance.UID[0:4]),
+				IPAdress:          ip,
+				NetworkDataSecret: networkDataSecretName,
+			},
+		)
+
+		sts = append(sts, common.Template{
+			Name:           fmt.Sprintf("%s-%s-networkdata", instance.Name, hostname),
+			Namespace:      instance.Namespace,
+			Type:           common.TemplateTypeConfig,
```
Comment (abays) [inline controllers/controllervm_controller.go:172]: Same question here regarding `common.TemplateTypeNone`.

File: controllers/controllervm_controller.go (hunk 1 above)
Comment (stuggi) [reply controllers/controllervm_controller.go:130]: yes, good catch, thanks!

File: controllers/controllervm_controller.go (hunk 2 above)
Comment (stuggi) [reply controllers/controllervm_controller.go:172]: yes, good catch, thanks!

**Review by abays — COMMENTED** [review]
/lgtm

**Pull request merged**